packages/components/src/pagination-control/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { __, sprintf } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { times } from 'lodash';
+import './style.scss';
+
+interface Props {
+	currentPage: number;
+	numberOfPages: number;
+	setCurrentPage: ( page: number ) => void;
+	classNames?: string | string[];
+}
+
+const PaginationControl: React.FunctionComponent< Props > = ( props ) => {
+	const { currentPage, numberOfPages, setCurrentPage, classNames, children } = props;
+
+	const classes = classnames( 'pagination-controls', classNames?.toString()?.split( ',' ) );
+
+	return (
+		<ul className={ classes } aria-label={ __( 'Guide controls' ) }>
+			{ times( numberOfPages, ( page ) => (
+				<li key={ page } aria-current={ page === currentPage ? 'page' : undefined }>
```

Comment:
If the number of pages changes, this key will break.

Say you deleted one page, all elements will shift left, meaning after deletion, the second element will get key=0 making React think it's the same as the now-deleted first element (since it has the same key). I'd make the length part of the key to fix that:

```suggestion
				<li key={ `${numberOfPages}-${page}` } aria-current={ page === currentPage ? 'page' : undefined }>
```

Reply:
👍